code/src/functions/install_initial_domain_mapping/index.ts

```diff
@@ -0,0 +1,15 @@
+import { AirdropEvent, installInitialDomainMapping } from '@devrev/ts-adaas';
+
+import initialDomainMapping from '../extraction/initial_domain_mapping.json';
+
+const run = async (events: AirdropEvent[]) => {
+  for (const event of events) {
+    try {
+      await installInitialDomainMapping(event, initialDomainMapping);
+    } catch (error) {
+      console.error('Failed to install initial domain mappings', error);
```

Comment:
The code expects that in case there is an error with installing initial domain mappings, we will log the error and return. Can the ADaaS snap-in function in such case?

Reply:
Extraction part is independent of this. However mappings won't be populated properly. @B-Lorentz what should we do in such a case?

Reply:
IDK honestly.
The snapin may still somewhat function, as many mappings can be provided by end-user.
But it is certainly a 'bad place to be'

Reply:
Is there a way to surface this error to the enduser?

Reply:
Yes, you can return error from the function itself.